### DISCUSSION TITLE
[Feature] Custom Airports

### DISCUSSION THREAD

**Discussion by palitu** [johanohly/AirTrail#26] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would also like to track some of the smaller regional flights, which are not in the DB that you utilise, (for good reason!)

something like Port Vila international airport (Bauerfield) to [Lonorore Airport](https://maps.app.goo.gl/TkbxiuUo1faPCYoD7).

This could be as simple as a Name, optional codes and Lat/Long with potentially more details as needed. 

It may also be worth denoting them as different types of airports on the map, with different colours and/or smaller circles.

cheers

**Comment by johanohly** [comment]
This was added in v2.0.0.
